README.md

```diff
@@ -1,69 +1,105 @@
+An efficient software development process is vital for success in building
+business applications on SAP Cloud Platform or SAP on-premise platforms.
+SAP addresses this need for efficiency with project "Piper". The goal of project
+"Piper" is to substantially ease setting up continuous deployment processes for
+the most important SAP technologies by means of Jenkins pipelines.
 
 Project "Piper" consists of two parts:
 
- * A [shared library][piper-library] containing steps and utilities that are required by Jenkins pipelines.
- * A set of [Jenkins pipelines][piper-pipelines] using the piper library to implement best practice processes.
+ * A [shared library][piper-library] containing steps and utilities that are
+   required by Jenkins pipelines.
+ * A set of [Jenkins pipelines][piper-pipelines] using the piper library to
+   implement best practice processes.
 
 ## What you get
 
-This repository contains Jenkins pipeline instructions in form of a `Jenkinsfile` (together with some auxiliary files) which require the [shared library][piper-library] of project "Piper" to run. They are available under `pipelines/<scenario>/Jenkinsfile` in this repository.
+This repository contains Jenkins pipeline instructions in form of a
+`Jenkinsfile` (together with some auxiliary files) which require the [shared
+library][piper-library] of project "Piper" to run. They are available under
+`pipelines/<scenario>/Jenkinsfile` in this repository.
 
-These best practice Jenkinsfiles are based on the general concepts of [Pipelines as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the Jenkins community at hand to optimize your continuous delivery processes.
+These best practice Jenkinsfiles are based on the general concepts of [Pipelines
+as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the
+Jenkins community at hand to optimize your continuous delivery processes.
 
-You can run the Jenkinsfiles out of the box or take them as a starting point for project-specific adaptations.
+You can run the Jenkinsfiles out of the box or take them as a starting point for
+project-specific adaptations.
 
-## Installation
+# Requirements
 
-Prerequisites:
-
-* [Installed shared library][piper-library-installation] of project "Piper".
+* [Installed shared library][piper-library-readme] of project "Piper".
 * Jenkins User with privileges to create jobs.
 
-There are two recommended ways to consume our `Jenkinsfile`. For both ways, you need to create a new pipeline job in Jenkins that [retrieves a pipeline from an SCM][jenkins-doc-pipelineFromSCM].
-The location of the `Jenkinsfile` may differ:
+# Download and Installation
+
+There are two recommended ways to consume our `Jenkinsfile`. For both ways, you
+need to create a new pipeline job in Jenkins that [retrieves a pipeline from an
```

Comment:
IMO it would be helpful with a screenshot of the Jenkins settings here, showing how to retrieve a pipeline from an SCM. The description of jenkins.io is not as helpful as a screenshot would be.

Reply:
Done